sdk/src/main/java/com/meniga/sdk/webservices/requests/AddComment.java

```diff
@@ -0,0 +1,21 @@
+package com.meniga.sdk.webservices.requests;
+
+import java.util.List;
+
+public class AddComment extends QueryRequestObject {
+
+	public List<Long> transactionIds;
+	public String comment;
+
+	public AddComment(List<Long> transactionIds, String comment) {
+		this.transactionIds = transactionIds;
+		this.comment = comment;
+	}
+
+	@Override
+	public long getValueHash() {
+		int result = transactionIds.hashCode();
```

Comment:
Not sure if this is enough?

Reply:
You can use kotlin here like in [GetBudgetRules.kt](https://github.com/meniga/mobile-sdk-android/blob/master/sdk/src/main/java/com/meniga/sdk/webservices/budget/GetBudgetRules.kt)

Reply:
Done